src/store/trees/trees.js

```diff
@@ -184,7 +184,6 @@ export default {
     treeUpdatedAt: ({tree}) => tree.updatedAt.date,
 
     supportedModes: ({ui}) => ui.supportedModes || [],
-    defaultModes: ({ui}) => ui.defaultModes || [],
```

Comment:
What's the reason we're removing defaultModes support?

Reply:
This change was made by me.
There was 'remove all default modes' in the description.
I suppose it meant remove them in our voto customization in the `flowBuilder.js` only?
LIke:
```
platformBuilderConfig.ui.defaultModes = []
```
@safydy-r Let's revert that commit (having the defaultModes changes) then